Ginger/GingerCoreCommon/GeneralLib/General.cs

```diff
@@ -81,5 +82,18 @@ public static bool IsFieldToAvoidInVeFieldSearch(string fieldName)
             }
 
         }
+        public static void AddOrUpdateInputParamValue(string Param, string Value, ObservableList<ActInputValue> actInputValue)
+        {
```

Comment:
we said this function is not needed here because you will use dummy Action for it, no?